docs/result_links_imports.md

```diff
@@ -27,20 +25,20 @@ GOOGLE_SHEET_ID="a-google-sheet-id"
 3. Run this take task:
 
 ```
-bundle exec rake content:import_locale_links_from_google_sheet
+bundle exec rake content:extract_results_links
```

Comment:
I think this should be:

```
 bundle exec rake content:export_results_to_csv
```

However this task now fails as we allow strings in the YAML as a result of [this PR](https://github.com/alphagov/govuk-coronavirus-find-support/pull/341) :*( 

Should we just change the rake command here and then make fixing the export a separate Trello task?

Reply:
Ah good spot, thx, I'll update

Reply:
I've rebased for the earlier YAML fix and corrected the bundle command in both places.